rmgpy/data/kinetics/family.py

```diff
@@ -3798,11 +3798,15 @@ def fix_labels_mol(mol, root_labels):
             if estimate_thermo:
                 for j, react in enumerate(r.item.reactants):
                     if rxns[i].reactants[j].thermo is None:
-                        rxns[i].reactants[j].thermo = tdb.get_thermo_data(react)
+                        therm_spc = deepcopy(rxns[i].reactants[j])
```

Comment:
Does it matter if you use `react` or `rxns[i].reactants[j]`? If not, could you just loop over `rxns[i].reactants`?